docs/FAQ.md

```diff
@@ -0,0 +1,106 @@
+
+### TensorRT/CUDA vs DirectML? 
+DirectML is the hardware-accelerated DirectX 12 library for machine learning on Windows and supports all DirectX 12 capable devices (Nvidia, Intel, AMD). This means that if you are targeting Windows GPUs, using the DirectML Execution Provider is likely your best bet. This can be used with both the ONNX Runtime as well as [WinML APIs](./WinRT_API.md).
+
+## Do the GPU builds support quantized models?
+The default CUDA build does not support any quantized operators right now. The TensorRT EP has limited support for INT8 quantized ops. In general, support of quantized models through ORT is continuing to expand on a model-driven basis. For performance improvements, quantization is not always required, and we suggest trying alternative strategies to [performance tune](./ONNX_Runtime_Perf_Tuning.md) before determining that quantization is necessary.
+
+## Why is the GPU build slower than CPU?
+Depending on which execution provider you're using, it may not have full support for all the operators in your model. Fallback to CPU ops can cause hits in performance speed. Moreover even if an op is implemented by our CUDA execution provider, we may not necessarily assign/place the op to the CUDA EP due to performance reasons. To see the placement decided by ORT, turn on verbose logging and look at the console output.
+
+See:
+* [Issue 1368](https://github.com/microsoft/onnxruntime/issues/1368)
+* [Issue 1246](https://github.com/microsoft/onnxruntime/issues/1246)
+
+## My converted Tensorflow model is so slow - why?
+
+NCHW and NHWC are two different memory layout for 4-D tensors.
+
+Most TensorFlow operations used by a CNN support both NHWC and NCHW data format. The Tensorflow team suggests that on GPU NCHW is faster but on CPU NHWC is sometimes faster in Tensorflow. However, ONNX only supports NCHW. As a result, if the original model is in NHWC format, when the model is converted extra transposes may be added. The latest [keras-onnx converter](https://github.com/onnx/keras-onnx) can remove many of these transposes. If you can, consider retraining the model using NCHW.
+
+## Does the Python API add any overhead to performance?
+The Python API doesn't add any overhead to performance when using the CPU execution provider only. When using the GPU provider, inputs and outputs need to be copied from CPU to GPU and vice-versa and the Python API doesn't allow inputs/outputs to be setup for this to be done prior to execution (calling run). Hence this copy needs to be done on the fly as part of execution. This eats into the execution time thereby impacting performance.
+
+## Why is ORT slower than PyTorch when using GPU?
+Refer to above answer. Also see: 
+* [Issue 2404](https://github.com/microsoft/onnxruntime/issues/2404#issuecomment-554593653)
```

Comment:
We shouldn't link to the issues - it makes it seem like it's an unresolved problem, and also the issues are long and require user to read through them. Better to summarize the tip/workaround/recommendation.

Reply:
For this we don't have any good recommendation until IOBinding support is added to the Python API; it's already in PR, so may be for now, for the answer to this question let's just say - "Refer to the answer above. We're actively working on supporting IOBinding in the Python API that allows copying of inputs to the GPU prior to calling Run."